components/ws-manager-mk2/controllers/status.go

```diff
@@ -111,6 +111,7 @@ func (r *WorkspaceReconciler) updateWorkspaceStatus(ctx context.Context, workspa
 
 	if failure != "" && !wsk8s.ConditionPresentAndTrue(workspace.Status.Conditions, string(workspacev1.WorkspaceConditionFailed)) {
 		// workspaces can fail only once - once there is a failed condition set, stick with it
+		log.Info("workspace failed", "workspace", workspace.Name, "reason", failure)
```

Comment:
Maybe Warn instead of Info?

Reply:
The logging framework only allows logging of info and errors.